code/parse/sexp.cpp

```diff
@@ -14750,9 +14782,32 @@ void sexp_destroy_instantly(int n)
 			ship_obj_p = &Objects[Ships[ship_num].objnum];
 
 			//if its the player don't destroy
-			if (ship_obj_p == Player_obj)
-				continue;
-			ship_destroy_instantly(ship_obj_p,ship_num);
+			if (ship_obj_p != Player_obj) {
+				ship_destroy_instantly(ship_obj_p, ship_num);
+
+				// multiplayer callback
+				Current_sexp_network_packet.start_callback();
+				Current_sexp_network_packet.send_ship(ship_num);
+				Current_sexp_network_packet.end_callback();
```

Comment:
Just to make sure I understand this correctly, if `Current_sexp_network_packet.start_callback()` is called multiple times then it basically sends multiple SEXP execution packets?

The reason I'm asking this is to make sure that this also works when specifying multiple ships.

Reply:
It does.  But you're right, I should go back and modify that a bit.